scripts/hail_batch/hgdp1kg_tobwgs_densify_new_variants/README.md

```diff
@@ -0,0 +1,9 @@
+# Densify TOB-WGS data after filtering for new variants
+
+This runs a Hail query script in Dataproc using Hail Batch in order to densify the TOB-WGS matrix after filtering for new variants. To run, use conda to install the analysis-runner, then execute the following command:
```

Comment:
```suggestion
This runs a Hail query script in Dataproc using Hail Batch in order to densify the TOB-WGS matrix before filtering for new variants. To run, use conda to install the analysis-runner, then execute the following command:
```

Reply:
I know what you mean by "new variants", but out of context that's fairly unclear. I wonder if we can come up with a better naming scheme for this new set of markers?

Reply:
Yes, I was actually thinking that, because 'new' can keep changing based off of what iteration we're on. I'll add something a bit more specific to this analysis in the README